index.js

```diff
@@ -49,10 +65,15 @@ function filterProductsByPriceRange(products, min, max) {
   If any errors occur in this function, it should return `0`.
 */
 function getTotalOfAllProductsByPriceRange(products, min, max) {
-  const filteredProducts = filterProductsByPriceRange(products, min, max);
-  const total = getCartTotal(filteredProducts);
-
-  return total;
+let total;
```

Comment:
Hey Mukayila! You can return total inside of the try block. If an error is encountered on line 70 or 71, it will go straight into the catch block, bypassing the rest of the try block.